app/views/dashboard.scala.html

```diff
@@ -1,11 +1,15 @@
 @()(implicit messages: Messages)
 
 @main(Messages("dashboard.header")) {
-<h1 class="govuk-heading-xl">@Messages("dashboard.title")</h1>
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
```

Comment:
What's the purpose of the two-thirds column on this page?

Reply:
The purpose of it is to keep it consistent to the other pages and to avoid the content to be wider than we want (in this case only the heading, but it might affect more in future depending on what we add to the page).

Reply:
Ah, I didn't realise it was the same on other pages. Do you think it's necessary? There are a few things in our basic page templates that were copied from elsewhere without really thinking about it.

Reply:
Yes it is actually really important because users don't read when the text is getting to long. There are numerous studies on this and GDS included it here https://design-system.service.gov.uk/styles/layout/#common-layouts and it says "For most types of page, we recommend using either a ‘two-thirds’ or a ‘two-thirds / one third layout’. That stops lines of text getting so long that the page becomes difficult to read on desktop devices." :-)

Reply:
Great, thanks - just wanted to make sure it was an intentional change, not something we'd copied from somewhere else without considering!